source/kraken/configuration.cpp

```diff
@@ -92,7 +92,9 @@ po::options_description get_options_description(const boost::optional<std::strin
         ("BROKER.vhost", po::value<std::string>()->default_value("/"), "vhost for rabbitmq")
         ("BROKER.exchange", po::value<std::string>()->default_value("navitia"), "exchange used in rabbitmq")
         ("BROKER.rt_topics", po::value<std::vector<std::string>>(), "list of realtime topic for this instance")
-        ("BROKER.timeout", po::value<int>()->default_value(100), "timeout for maintenance worker in millisecond")
+        ("BROKER.timeout", po::value<int>()->default_value(200), "timeout for maintenance worker in millisecond")
+        ("BROKER.max_batch_nb", po::value<int>()->default_value(5000), "max size of the realtime message")
+        ("BROKER.retrieving_timeout", po::value<int>()->default_value(10000), "max duration the worker is going to spend when retrieving messages")
```

Comment:
Trying to clarify the naming a bit here.

Also, would probably lower the value, as it's most of the time OK already with a 100ms timeout.
But maybe your tests showed opposite conclusion, in this case sorry :pray: 

```suggestion
        ("BROKER.retrieving_total_timeout", po::value<int>()->default_value(1000), "max total duration the worker is going to spend when retrieving messages")
```

Reply:
1s is just too few given the size of messages from kirin, a message may take several ms to be polled from rabbitmq.